win_configure/projects.VS2008/gen_server_ecl.bat

```diff
@@ -16,7 +16,7 @@ echo "delete file not a success"
 echo. >  "../../src/lib/Libattr/job_attr_def.c"
 )
 echo "Parsing the Master file for server and ecl"
-"C:\Program Files\Python27\python.exe" "../../buildutils/attr_parser.py" -m "../../src/lib/Libattr/master_job_attr_def.xml" -s "../../src/lib/Libattr/job_attr_def.c" -e "../../src/lib/Libecl/ecl_job_attr_def.c" -a job
+"..\..\..\binaries\python\python.exe" "../../buildutils/attr_parser.py" -m "../../src/lib/Libattr/master_job_attr_def.xml" -s "../../src/lib/Libattr/job_attr_def.c" -e "../../src/lib/Libecl/ecl_job_attr_def.c" -a job
```

Comment:
The exe path follows windows style where as other parameters follow unix style. Please make it uniform.